mslib/msui/msui.py

```diff
@@ -294,10 +295,18 @@ def get_shortcuts(self):
 
         return shortcuts
 
-    def filter_shortcuts(self, text):
+    def filter_shortcuts(self, text="default", rerun=True):
         """
```

Comment:
I agree to this change, but the word "default" may be better replaced by something like "Not Available", "Nothing" as in the test used.

The word "default" is used in some string definitions in ui_tableview_window.ui and others. It also impresses "a default you know" so that would need a docstring explaining "default" is something not existing.

Reply:
Done